library/zookeeper

```diff
@@ -10,6 +10,10 @@ Tags: 3.8.2, 3.8
 GitCommit: 952b2f523f4f9a829d77114ddbe08ab9249d820b
 Directory: 3.8.2
 
-Tags: 3.8.2-temurin, 3.8-temurin, latest
+Tags: 3.8.2-temurin, 3.8-temurin
 GitCommit: 952b2f523f4f9a829d77114ddbe08ab9249d820b
 Directory: 3.8.2
+
+Tags: 3.9.0, latest
```

Comment:
I'll go ahead and merge, but did you also want `3.9.0-temurin, 3.9-temurin` tags too?

🤔 It seems that the `3.8.2, 3.8` section uses the same source commit and directory as `3.8.2-temurin, 3.8-temurin`, is that correct? The sections could be combined, but the `bashbrew` tool detects the duplication and combines them anyway.

Reply:
No I didn't wan't `temurin` tags. I follow the plan described here: https://github.com/docker-library/official-images/pull/12949#issuecomment-1383656252

> Yes, the plan is to move the plain tags during corresponding version upgrades. For example, currently 3.5.9 and 3.5 are based on openjdk. Once 3.5.10 is released 3.5.10 and 3.5 becomes temurin

Good catch about `3.9`. Thanks!